web/packages/core/src/ruffle-player.ts

```diff
@@ -455,6 +457,7 @@ export class RufflePlayer extends HTMLElement {
             this.container.style.visibility = "";
         }
 
+        this.isPreloaderVisible = false;
```

Comment:
Move this here: https://github.com/ruffle-rs/ruffle/blob/4f05d1852dd6c8e654aead0dcff7fefbc07264c9/web/packages/core/src/ruffle-player.ts#L1407